appservice/query/query.go

```diff
@@ -217,7 +217,7 @@ func (a *AppServiceQueryAPI) Locations(
 		}
 
 		if err := requestDo[[]api.ASLocationResponse](as.HTTPClient, url+"?"+params.Encode(), &asLocations); err != nil {
-			log.WithError(err).Error("unable to get 'locations' from application service")
+			log.WithError(err).Error("unable to get 'locations' from application service " + as.ID)
```

Comment:
Can we use `WithField("application_service", as.ID)` instead? 
E.g.
```suggestion
			log.WithError(err).WithField("application_service", as.ID).Error("unable to get 'locations' from application service")
```